src/test/FuzzerImpl.cpp

```diff
@@ -775,6 +884,76 @@ TransactionFuzzer::xdrSizeLimit()
     return 50 * FuzzUtils::FUZZER_MAX_OPERATIONS;
 }
 
+// Given an Operation generated with autocheck through the XDR generator, try in
+// operation-type-specific ways to constrain the ranges of parameters into
+// spaces that will make it practical for the fuzzer to explore, while
+// preserving as much of the randomness as we can within the constrained space,
+// and occasionally exploring outside the constrained space to search for bugs
+// that would only be triggered by larger parameter values.
+void
+TransactionFuzzer::constrainOperation(Operation& op)
```

Comment:
This change is out of scope for this PR. This seems to be loosely related to `The data generated to seed the fuzzer via the gen-fuzz is very low quality` which is much further down in the list of things to do.

Reply:
> This change is out of scope for this PR. This seems to be loosely related to The data generated to seed the fuzzer via the gen-fuzz is very low quality which is much further down in the list of things to do.

Yes, that's what I was trying to start on.  I've moved that commit out of this PR and into a new branch.